pkg/controller/elasticsearch/driver/upscale_state_test.go

```diff
@@ -80,7 +80,7 @@ func Test_upscaleState_limitNodesCreation(t *testing.T) {
 		{
 			name:        "upscale masters from 3 to 4, but no creates allowed: should limit to 0",
 			state:       &upscaleState{allowMasterCreation: true, isBootstrapped: true, createsAllowed: pointer.Int32(0)},
-			actual:      sset.TestSset{Name: "sset", Replicas: 3, Master: true}.Build(),
+			actual:      sset.TestSset{Name: "sset", Replicas: 3, Master: false}.Build(),
```

Comment:
Do we need to change `Master` to `false` here ? Is it something left from a previous change ?

Reply:
:o indeed, no reason for that change!